desktop-app/src/renderer/components/Previewer/Device/index.tsx

```diff
@@ -282,6 +282,15 @@ const Device = ({ isPrimary, device, setIndividualDevice }: Props) => {
       webview.removeEventListener('did-fail-load', didFailLoadHandler);
     });
 
+    if (device.isMobileCapable) {
+      webview.addEventListener('dom-ready', () => {
```

Comment:
@manojVivek you are right. My bad,  I thought I have removed it since I moved it into a separate effect.
Thanks for pointing it out.
I will remove it then